examples/hybrid/sphere/deformation_flow.jl

```diff
@@ -197,19 +201,19 @@ function rhs!(dydt, y, (coords, face_coords), t)
     cuvw = Geometry.Covariant123Vector.(uₕ) .+ Geometry.Covariant123Vector.(cw)
 
     @. dρq1 -= hdiv(cuvw * ρq1)
-    @. dρq1 -= vdivf2c(w * Ic2f(ρq1))
+    @. dρq1 -= vdivf2c(third_order_upwind_c2f.(w, ρq1 ./ ρ))
```

Comment:
I think you need to multiply by ρ:
```suggestion
    @. dρq1 -= Ic2f(ρ) * vdivf2c(third_order_upwind_c2f.(w, ρq1 ./ ρ))
```

Reply:
Oh, ok. Thanks. I had tried both versions in the two commit. One with `ρ` inside and the other one outside, but I mistakenly did not add it outside. Will do now. Thanks

Reply:
@simonbyrne wait, actually, that would give a `LoadError: Broacasted spaces are not the same ClimaCore.Spaces type` error. `ρ` is already at the center and we want the operation to land at centers

Reply:
So it will have to be just a simple multiplication by `ρ`, without the interpolation operator

Reply:
@simonbyrne I did that in commit [514aa26](https://github.com/CliMA/ClimaCore.jl/pull/714/commits/514aa261cc0f122cb808968d2f8c898d9a248d06) and I still see the same error with roughly the same magnitude, for same `dt` and resolution. 

```
Test Failed at /home/valeria/CliMA/ClimaCore.jl/examples/hybrid/sphere/deformation_flow.jl:244
  Expression: ≈(q1_error, 0.0, atol = 0.7)
   Evaluated: 0.7360619443393059 ≈ 0.0 (atol=0.7)
```